### PR TITLE
Terrain & ore overlay minimap color customization

### PR DESCRIPTION
**Pull request by Starkku**
- TerrainTypes & Tiberiums can now be made to display on minimap with different colors by setting `MinimapColor`.

In `rulesmd.ini`:
```ini
[SOMETERRAINTYPE]  ; TerrainType
MinimapColor=      ; integer - Red,Green,Blue

[SOMEORE]          ; Tiberium
MinimapColor=      ; integer - Red,Green,Blue
```

Additionally `RadarInvisible`, if set to true, will now hide TerrainTypes from displaying on minimap at all. Because `RadarInvisible` defaults to true for TerrainTypes and it is unused by the game normally, I have taken a liberty of overriding this default to false.

### CODE REVIEW

**Comment by FS-21** [comment]
(regarding Ores) I tried different colors in a map where I use 4 tiberiums and works, but the loading screen still paint in yellow and purple.
imagen

**Comment by Starkku** [comment]
Loading screen preview is not generated by game, but is image data embedded in the map file generated by an external source like map editor.

**Comment by FS-21** [comment]
Now I tried the TerrainType test, I modified all trees with the same shiny light blue color but doesn't work.

[TREE01]
Name=Tree #01
RadarColor=0,192,0
TemperateOccupationBits=4
SnowOccupationBits=6
IsFlammable=yes
Armor=wood
Strenght=200
MinimapColor=0,255,255

imagen

**Comment by Starkku** [comment]
Should be fixed now, it was because `RadarInvisible` defaulted to true on TerrainTypes, which previously didn't matter because it wouldn't be used anywhere. I've opted to change the default to false.

**Comment by FS-21** [comment]
The last binary fixed the radar color in trees.

**Comment by mevitar** [comment]
Appears to work fine, and undefined values make the game revert back to default colors.

File: Phobos.vcxproj [hunk 1]
```diff
@@ -144,6 +146,7 @@
     <ClInclude Include="src\Ext\TechnoType\Body.h" />
     <ClInclude Include="src\Ext\Techno\Body.h" />
     <ClInclude Include="src\Ext\TerrainType\Body.h" />
+	<ClInclude Include="src\Ext\Tiberium\Body.h" />
```
Comment (Metadorius) [inline Phobos.vcxproj:149]: Seems to have inconsistent tabs/spaces.

**Review by Metadorius — APPROVED** [review]

**Pull request merged**